### ISSUE TITLE
no-unused-variable when re-exporting

### ISSUE THREAD

**Issue by ShadowManu** [palantir/tslint#913]
Given the following code-snippet:

```javascript
import aDefaultImport from './some-module';

export {
  aDefaultImport
}
```
tslint gives the `no-unused-variable` linter error. Shouldn't be that way AFAIK.

**Comment by JKillian** [comment]
Thanks for the report @ShadowManu! This is a duplicate of #684 which should be fixed in the next nightly version of TypeScript
